pkgs/development/misc/haskell/hercules-ci-optparse-applicative.nix

```diff
@@ -0,0 +1,21 @@
+{ mkDerivation, ansi-wl-pprint, base, fetchgit, lib, process, QuickCheck
+, transformers, transformers-compat
+}:
+mkDerivation {
+  pname = "hercules-ci-optparse-applicative";
+  version = "0.16.1.0";
+  src = fetchgit {
```

Comment:
```suggestion
  version = "0.16.1.0";

  src = fetchgit {
```

Reply:
That does look nicer, but it's generated code. I'd rather avoid the unnecessary diff lines next time I generate it. (Though hopefully I can remove it by then)